playwright/mockData/orgs/KPD/campaigns/ReferendumSignatures/index.ts

```diff
@@ -0,0 +1,22 @@
+import KPD from '../..';
+import { ZetkinCampaign } from '../../../../../../src/types/zetkin';
```

Comment:
Any chance we could use the global paths here, the way that we did in the application codebase?

Reply:
This is what would be required (https://playwright.dev/docs/test-typescript), which I'm not sure is worth the extra steps... We could make it only require one script of course, but I wonder if it's worth it... What do you think @richardolsson

Reply:
I see the problem. I think let's leave it as it is for now at least. What do you think?